### PR TITLE
Clarify expected return value of first assignment (in Recursion Project lesson)

### PR DESCRIPTION
**Pull request by Eduardo06sp**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).

#### 1.Describe the changes made:

I reworded the first assignment a bit from:
> Write a method `#fibs` which takes a number and returns that many members of the fibonacci sequence. Use iteration for this solution.

to 

> Using iteration, write a method `#fibs` which takes a number and returns that many members of the fibonacci sequence. It should return an array.

This is to make it clear that it is not the exact same solution as the ones provided in the Code Quiz on Recursion nor the Khan Academy fibonacci walk-through under Additional Resources (in the previous lesson).

### CODE REVIEW

File: ruby_programming/computer_science/project_recursion.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The [Fibonacci Sequence](http://en.wikipedia.org/wiki/Fibonacci_number), which s
 ### Assignment 1
 
 <div class="lesson-content__panel" markdown="1">
-  1. Write a method `#fibs` which takes a number and returns that many members of the fibonacci sequence.  Use iteration for this solution.
+  1. Using iteration, write a method `#fibs` which takes a number and returns that many members of the fibonacci sequence. It should return an array.
```
Comment (wise-king-sullyman) [inline ruby_programming/computer_science/project_recursion.md:8]:
```suggestion
  1. Using iteration, write a method `#fibs` which takes a number and returns an array containing that many numbers from the fibonacci sequence.
```

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Thank you for taking the time to do this PR!

You've definitely moved this in a better direction, but I'm requesting a small change in ordering/wording used.

The change I've suggested gets the input and output type closer together, and moves the key `from the fibonacci sequence` part to the end where it should be less likely that people read part of the instruction and jump to **doing** with the incorrect assumption that they know what the rest of the sentence will say. It also changes the `members of` phrasing to something that feels clearer to me.

File: ruby_programming/computer_science/project_recursion.md (hunk 1 above)
Comment (Eduardo06sp) [reply ruby_programming/computer_science/project_recursion.md]: That makes perfect sense, thank you. I made the change.

**Review by wise-king-sullyman — APPROVED** [review]
Awesome! I think these changes will clear up confusion for a lot of people! :)

**Pull request merged**